architecture/additional_concepts/authentication.adoc

```diff
@@ -685,3 +685,186 @@ responses from `/oauth/authorize`, and how they should be handled:
 * `openshift_auth_form_password_count` counts the number of web console login attempts.
 * `openshift_auth_form_password_count_result` counts the number of web console login attempts by result (success or error).
 * `openshift_auth_password_total` counts the total number of `oc login` and web console login attempts.
+
+[[authentication-manage-oauth]]
+== Managing OAuth Objects
+
+Each user has an individual directory where OAuth Client Authorizations for that user are stored:
+*_/openshift.io/oauth/clientauthorizations/<userName>/<clientName>_*.
+
+Having a specific directory allows for faster lookup times for a user. 
+Previously, {product-title} maintained OAuth Client Authorizations in etcd.
```

Comment:
Umm, everything is still in etcd.  Nothing is stored locally or in some file system.